docs/source/api.rst

```diff
@@ -59,3 +58,26 @@ There's also a HAWC client available in R:
         ids=hero_ids
     )
 
+The API for the R-client is roughly a mirror of the Python client, but may be lagging in features.
```

Comment:
Did you mean lagging or lacking? If lagging maybe "behind in features", I think that may read better but that's just personal opinion.

Reply:
thanks updated the text